src/grafana/chart/templates/uds-package.yaml

```
@@ -28,23 +28,20 @@ spec:
         targetPort: 3000
 
     allow:
-      - direction: Ingress
+      # Egress allowed to Loki
+      - direction: Egress
         selector:
           app.kubernetes.io/name: grafana
-        remoteNamespace: tempo
+        remoteNamespace: loki
         remoteSelector:
-          app.kubernetes.io/name: tempo
-        port: 9090
-        description: "Tempo Datasource"
+          app.kubernetes.io/name: loki
+        description: "for loki"
```

very nit, but might prefer this description to be "Loki Datasource"
```suggestion
        description: "Loki Datasource"
```

Current netpol name for reference:
```
allow-grafana-egress-for-loki                            app.kubernetes.io/name=grafana   8s
```

---

I think we could also lock this policy down to a specific port - potentially port 80 (https://github.com/defenseunicorns/uds-core/blob/lock-down-egress/src/grafana/chart/templates/datasources.yaml#L21)